platform/admin/core/admin-core-logviewer/src/main/webapp/js/components/log-viewer/log-viewer.js

```diff
@@ -73,10 +75,22 @@ export default ({ dispatch, displaySize, logs, filter }) => {
     }
   }
 
+  const getTableClasses = () => {
+    if (expandedHash === undefined) {
+      return 'table'
+    } else {
+      return 'table dimUnselected'
+    }
+  }
+
+  const deselect = () => {
+    dispatch(actions.expandEntry(undefined))
+  }
+
   return (
     <div className='container'>
       <div className='filterRow'>
-        <table className='table'>
+        <table className='table' onClick={deselect}>
```

Comment:
If the user clicks an entry, it expands the entry and dims the surrounding entries. If after clicking, the user uses the filters, it will not deselect the entry, and all logs returned by the filter will be dimmed(except for the selected log if it passes the filters). the user would then have to select a new log, and then click it again to deselect it in order to undim everything.

An onClick event was added to the table, so if a user clicks on anywhere in the filter area, it deselects all logs. It gives it more of a "focused item" feel instead of a rigid toggle that persists even through filtering, so if the user "clicks-off" of the item, it deselects it. It also makes sure that the expanded item is deselected when filtering.